### PR TITLE
Add PreviousFileName to the PullRequestFile model

### LINKED ISSUE #1768: Add PreviousFileName property for PullRequestFile
When a file is renamed the github api returns a previous_filename property on [pull request files](https://developer.github.com/v3/pulls/#list-pull-requests-files). Could we add the property to PullRequestFile similar to how it's returned on [GithubCommitFile](https://github.com/octokit/octokit.net/blob/c9b2c1260bc87b7782d4fd9645d43a8885203923/Octokit/Models/Response/GitHubCommitFile.cs#L87)?

### PR DESCRIPTION
**Pull request by Kaneraz**
Fixes #1768 

Pretty straight forward change, and gets my consuming project rolling again. Unit test and convention tests pass, but it looks like I'm having unrelated issues with the integration tests.

### CODE REVIEW

File: Octokit/Models/Response/PullRequestFile.cs [hunk 1]
```diff
@@ -34,8 +35,10 @@ public PullRequestFile(string sha, string fileName, string status, int additions
         public string RawUrl { get; protected set; }
         public string ContentsUrl { get; protected set; }
         public string Patch { get; protected set; }
+        [Parameter(Key = "previous_filename")]
+        public string PreviousFileName { get; protected set; }
 
-        internal string DebuggerDisplay
+    internal string DebuggerDisplay
```
Comment (ryangribble) [inline Octokit/Models/Response/PullRequestFile.cs]: Something happened to whitespace/alignment here?

Comment (Kaneraz) [reply Octokit/Models/Response/PullRequestFile.cs]: Ah, sorry about that.

**Comment by ryangribble** [comment]
Many thanks @Kaneraz

**Comment by ryangribble** [comment]
release_notes: Add `PreviousFileName` field to `PullRequestFile` response

**Pull request merged**